tests/test_pump_scheduler.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import absolute_import
+import unittest
+import datetime
+
+import mock
+
+from greenpithumb import pump_scheduler
+
+
+class PumpSchedulerTest(unittest.TestCase):
+
+    def setUp(self):
+        self.mock_clock = mock.Mock()
+
+    def test_wake_hour_less_than_sleep_hour(self):
+        """Test rollover from one day to the next.
+        
+        If the sleep period spans two days, the pump should not run between 
+        midnight and the wake hour.
+        """
+        self.mock_clock.now.return_value = datetime.datetime(2016, 5, 24, 0)
+        sleep_windows = ((22, 8),)
```

Comment:
Oh sorry I'm stupid and changed the wrong brackets.

```python
sleep_windows = ([22,8])
```

just evaluates to:

```python
sleep_windows = [22,8]
```

Because it thinks the parens are for precedence. We want the other way around

```python
sleep_windows = [(22, 8)]
```